ytfzf

```diff
@@ -954,16 +996,31 @@ update_ytfzf () {
 	exit
 }
 
-sort_video_data_date () {
-	while IFS= read -r _VLine
+#gives a value to sort by (this will give the unix time the video was uploaded)
+if ! function_exists "data_sort_key"; then
+    data_sort_key () {
+	sort_by="${5#|}"
+	sort_by="${sort_by#Streamed}"
+	#print the data that should be sorted by
+	printf "%d" "$(date -d "${sort_by}" '+%s')"
+	unset sort_by
+    }
+fi
+#the function to use for sorting
+if ! function_exists "data_sort_fn"; then
+    data_sort_fn () {
+	sort -nr
+    }
+fi
+sort_video_data_fn () {
+	while IFS= read -r line
 	do
 		IFS="$tab_space"
-		set -- $_VLine
-		_Date=${5#|}
-		_Date=${_Date#Streamed}
-		printf "%d\t%s\n" "$(date -d "${_Date}" '+%s')" "$_VLine"
-	done | sort -nr | cut -f2-
-	unset IFS _VLine _Date
+		set -- $line
```

Comment:
since we are using `IFS` into a function `data_sort_key`, this `set` will not be getting used

Reply:
removed